src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -333,3 +333,91 @@ ov::pass::GeluFusionWithTanhNoPower::GeluFusionWithTanhNoPower() {
     auto m = std::make_shared<pattern::Matcher>(mul6, matcher_name);
```

Comment:
optional: it would be great if you would help to simplify `GeluFusionWithTanhNoPower` as well (moving constants checks to pattern, using gelu_replacer helper) if it's possible of course

Reply:
Applied, thanks!